test/contrib/rails/rack_middleware_test.rb

```diff
@@ -178,4 +183,22 @@ class FullStackTest < ActionDispatch::IntegrationTest
     assert_equal(request_span.get_tag('http.status_code'), '404')
     assert_equal(request_span.status, 0)
   end
+
+  test 'rails configuration enables distributed tracing' do
```

Comment:
The change in the `configure()` method has been done to make this test. Mostly my intention was to be sure we can do distributed tracing using Rails settings. Since we may not need this test later (Rack will be configured separately maybe?), we may rollback the last change in the future.